integration-tests/lambda-view.spec.ts

```diff
@@ -142,6 +144,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
 
       await op.confirmation()
       const contract = await op.contract();
+      console.log("lambda_view #2 contract address: "+ contract.address)
```

Comment:
I think you forgot to remove the console.log here

Reply:
fixed